OpenRA.Mods.Common/Traits/Health.cs

```diff
@@ -226,9 +226,9 @@ public void InflictDamage(Actor self, Actor attacker, Damage damage, bool ignore
 					self.Dispose();
 
 				if (attacker == null)
-					Log.Write("debug", "{0} #{1} was killed.", self.Info.Name, self.ActorID);
+					Log.Write("debug", "Tick {2}: {0} #{1} was killed.", self.Info.Name, self.ActorID, self.World.WorldTick);
```

Comment:
And these too. Do we have any other tick-based debug output?

Reply:
> Do we have any other tick-based debug output?

Not really. I guess I'll just remove it.